packages-exp/functions-exp/src/api.ts

```diff
@@ -55,6 +56,23 @@ export function getFunctions(
   return functionsInstance;
 }
 
+/**
+ * Modify this instance to communicate with the Cloudd Functions emulator.
+ *
+ * <p> Note: this must be called before this instance has been used to do any operations.
+ *
+ * @param host the emulator host (ex: localhost)
```

Comment:
Capitalize first letter of param descriptions ("The"), sorry for being pedantic, but it will go into docs. Not sure if devsite will automatically capitalize but exp docs don't go into devsite yet: https://github.com/firebase/firebase-js-sdk/blob/master/docs-exp/functions.getfunctions.md

Reply:
I don't think this change got in?

Reply:
Ohh sorry yeah I did miss this one.

Reply:
Now it's in.